datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/entity/versioning/LinkAssetVersionResolver.java

```diff
@@ -0,0 +1,88 @@
+package com.linkedin.datahub.graphql.resolvers.entity.versioning;
+
+import static com.linkedin.datahub.graphql.resolvers.ResolverUtils.bindArgument;
```

Comment:
Just heads up I would expect code coverage warnings to go away here with basic resolver unit tests. All new resolvers should ideally hve tests